src/Microsoft.Diagnostics.ExtensionCommands/FindEphemeralReferencesToLOHCommand.cs

```diff
@@ -0,0 +1,175 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Runtime;
+using static Microsoft.Diagnostics.ExtensionCommands.TableOutput;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    [Command(Name = "FindEphemeralReferencesToLOH", Help = "Finds ephemeral objects which reference the large object heap.")]
```

Comment:
The convention so far is all the managed commands be lower case especially new ones.   Also is there any way to make them shorter?  We don't support command completion (yet).

Reply:
I was hoping !sos was case-insensitive.  At minimum we shouldn't allow `Foo` and `foo` to be two different commands.  I'll find a shorter name though =)

Reply:
It has been suggested that commands are case-insensitive but both native debuggers we run under they are not.  For windbg/dbgeng we usually have aliases in sos.def for the commands.  For lldb and dotnet-dump the lower case version is the one that can be run directly without "sos <cmd>". 

I hope in the end, you do chose a all lowercase one for these new commands.